.github/actions/install_dependencies_qt/action.yml

```diff
@@ -0,0 +1,16 @@
+name: install_dependencies_qt
+description: Installs dependencies for qt
+
+inputs:
+  os:
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install Ubuntu dependencies
+      if: inputs.os == 'ubuntu-latest'
+      run: |
+        sudo apt-get update
+        sudo apt-get install libegl1
```

Comment:
Are we sure this is necessary?

Reply:
it was for pyside6. I can remove it and see what happens with pyqt6

Reply:
it did not work without libegl1